lib/watir/elements/element.rb

```diff
@@ -395,13 +376,13 @@ def wd
 
     #
     # Returns true if this element is visible on the page.
+    # Raises exception if element does not exist
     #
     # @return [Boolean]
     #
 
     def visible?
-      assert_exists
-      element_call { @element.displayed? }
+      element_call(:assert_exists) { @element.displayed? }
```

Comment:
None of the predicate methods wait by default. Only taking actions on the element uses an automatic wait.